### DISCUSSION TITLE
Problema esphome

### DISCUSSION THREAD

**Discussion by eremite1** [indomus/forum#2561] [GENERALE - Discussioni generiche]
Salve a tutti, ho un problema con esphome
Ho un regolatore di carica solare victron che ho integrato in home assistant tramite esphome con esp8266, fino a qualche giorno fa funzionava tutto, adesso però l'entità mi danno come risultato tutte "non disponibile", se vado però nell'integrazione esphome nonsolo risulta online ma se vado a fare il log le varie entità funzionano correttamente e mostrano anche i vari valori rilevati. Allego foto, inultimo metto anche il log di home assistante (ci sono diversi errori ma quelli che mi interessano ora sono quelli relativi a esphome e victron, gli altri li risolverò in seguito) ma non è che ci ho capito granchè, sapete aiutarmi?

**Comment by eremite1** [comment]
non so come caricare gli screen

**Comment by eremite1** [comment]
![log di sistema](https://github.com/indomus/forum/assets/140816116/465fd939-74d9-4dcb-85b4-caff826f1e67)
![log esphome](https://github.com/indomus/forum/assets/140816116/0dfbbc1c-d1f5-4f76-843c-fe5d581b4b55)
![sensori non disponibili](https://github.com/indomus/forum/assets/140816116/11dce2ef-b787-4530-aa83-f3061442bb12)

**Comment by KC-inDomus** [comment]
non devi caricare gli screen ma, come già detto, [I LOG DI HOME ASSISTANT](https://www.home-assistant.io/integrations/logger/) NEL MOMENTO IN CUI PERDE CONTATTO CON GLI HOST ESPHOME.

**Comment by eremite1** [comment]
questi?
`Logger: aioesphomeapi.reconnect_logic
Source: runner.py:179
First occurred: 20:41:13 (2 occurrences)
Last logged: 20:43:19

Can't connect to ESPHome API for victron-mppt @ 192.168.1.22: Error connecting to ('192.168.1.22', 6053): [Errno 113] Connect call failed ('192.168.1.22', 6053)`

`Logger: homeassistant.components.sensor
Source: helpers/entity_platform.py:554
Integration: Sensore (documentation, issues)
First occurred: 20:41:10 (40 occurrences)
Last logged: 20:43:16

Platform esphome does not generate unique IDs. ID victron-mppttext_sensorvictron-mppt_error already exists - ignoring sensor.victron_mppt_error
Platform esphome does not generate unique IDs. ID victron-mppttext_sensorvictron-mppt_tracking_mode already exists - ignoring sensor.victron_mppt_tracking_mode
Platform esphome does not generate unique IDs. ID victron-mppttext_sensorvictron-mppt_firmware_version already exists - ignoring sensor.victron_mppt_firmware_version
Platform esphome does not generate unique IDs. ID victron-mppttext_sensorvictron-mppt_device_type already exists - ignoring sensor.victron_mppt_device_type
Platform esphome does not generate unique IDs. ID victron-mppttext_sensorvictron-mppt_serial_number already exists - ignoring sensor.victron_mppt_serial_number`

`Logger: homeassistant.components.binary_sensor
Source: helpers/entity_platform.py:554
Integration: Sensore binario (documentation, issues)
First occurred: 20:41:10 (4 occurrences)
Last logged: 20:43:16

Platform esphome does not generate unique IDs. ID victron-mpptbinary_sensorvictron-mppt_load_state already exists - ignoring binary_sensor.victron_mppt_load_state
Platform esphome does not generate unique IDs. ID victron-mpptbinary_sensorvictron-mppt_relay_state already exists - ignoring binary_sensor.victron_mppt_relay_state`
